eurostag-step-up-transformer/src/test/java/eu/itesla_project/iidm/ddb/eurostag/EurostagDDBTest.java

```diff
@@ -0,0 +1,76 @@
+package eu.itesla_project.iidm.ddb.eurostag;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.FileSystem;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Test;
+
+import com.google.common.jimfs.Configuration;
+import com.google.common.jimfs.Jimfs;
+
+public class EurostagDDBTest {
+
+    @Test
+    public void testEurostagDDB() throws IOException, URISyntaxException {
+        try (FileSystem fs = Jimfs.newFileSystem(Configuration.unix())) {
+        Path path1 = fs.getPath("/path1");
```

Comment:
Path path1 = Files.createDirectory(fs.getPath("/path1"));

Same remark for the others directories, files and symbolic links